pkg/cloudprovider/aws/packing/nodecapacity.go

```diff
@@ -74,6 +77,41 @@ var (
 	}
 )
 
+func instanceTypeInfoToNodeCapacity(instanceTypeInfo ec2.InstanceTypeInfo) (*nodeCapacity, error) {
+	instanceTypeName := *instanceTypeInfo.InstanceType
+	vcpusInMillicores := fmt.Sprintf("%dm", *instanceTypeInfo.VCpuInfo.DefaultVCpus*1000)
```

Comment:
Are cores ever non-integer? Otherwise you can drop the M

Reply:
ah good point, yeah cores are always ints